### ISSUE TITLE
JetBrains Mono Medium displays in italics if Medium Italic is installed

### ISSUE THREAD

**Issue by zanedp** [JetBrains/JetBrainsMono#120]
If both Medium and Medium Italic are installed, when JetBrains Mono Medium is selected as the font, all characters are rendered in italics.

However, if JetBrains Mono Medium Italic is _not_ installed, when JetBrains Mono Medium is selected as the font, JetBrains Mono Medium is rendered as expected. This suggests that there's some property that is set incorrectly in one of those two font files in the 1.0.2 zip file.

I've seen this behavior in VS Code and Visual Studio 2019 on Windows 10 on two different computers.

Using version 1.0.2 of the TTF fonts.

**Comment by vadersb** [comment]
Same issue here. In Windows Fonts folder, if both Medium and Medium Italic are installed, Medium belongs to **JetBrains Mono** family by mistake, while Medium Italic belongs to **JetBrains Mono Medium** family. Probably that's why Medium Italic is used instead of Medium.

**Comment by J3Rik0** [comment]
Can confirm the issue is present in Windows, in Linux (Fedora with Gnome) the proper font is picked up when specifying the medium weight.

**Comment by rmaenk** [comment]
Confirm the same issue in Visual Studio 2017 (build 15.9.9)

**Comment by philippnurullin** [comment]
@zanedp Is it still present in v1.0.3? The family organisation was redone.

**Comment by zanedp** [comment]
I'm still seeing the same behavior with v1.0.3. (I even rebooted the machine after installing the new version, just in case.)

**Comment by philippnurullin** [comment]
@zanedp Sad to hear it. Ok, we will continue to look whats causing this problem. Sorry for the inconvenience.

**Comment by vadersb** [comment]
@philippnurullin , I'm getting different results, not as @zanedp does. Everything is fine on my machine (Windows 10, VS 2019, VS Code, etc) after installing v1.0.3. May suggest that @zanedp hasn't cleaned up the previously installed font (v1.0.2).

**Comment by philippnurullin** [comment]
@vadersb Thanks! Then it look like you described. The old files must be deleted first.

**Comment by zanedp** [comment]
@vadersb Is right:

Originally, I just installed the 1.0.3 fonts over the 1.0.2 fonts. Then, noticed it was still showing 1.0.2 for the Medium. I deleted that 1.0.2 Medium, and then it showed as version 1.0.3. I rebooted and then posted the comment above. However, after @vadersb's comment, I looked again, and all the fonts were still showing 1.0.3. But, I deleted them all. And then two more 1.0.2 versions popped up as still installed (and being hidden by 1.0.3). I deleted those. Then, I kid you not, some more appeared. Deleted those. (What the heck, Windows?!)

Now, I've installed 1.0.3 again, and everything is working as expected. I'm going to mark this defect as closed.
